2021-06-07 18:26:55,572 - __main__ - INFO - Error detected
Traceback (most recent call last):
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Создание_пассажиров\Сreate_passanger.py", line 31, in genlog
    self.create_passanger()
  File "C:\Work\Работа\Питон\COVID_19_ 9880\Создание_пассажиров\Сreate_passanger.py", line 50, in create_passanger
    driver.find_element_by_css_selector("#j_idt60 > div.nano.layout-tabmenu-nav > ul > li:nth-child(7) > a > div").click()
AttributeError: 'NoneType' object has no attribute 'click'
